### PR TITLE
hdfs url is now a configurable parameter in Confluent Connect

### PR DESCRIPTION
**Pull request by debasishg**

### CODE REVIEW

File: repo/packages/C/confluent-connect/7/resource.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "assets": {
     "container": {
       "docker": {
-        "image": "confluentinc/cp-kafka-connect:3.2.2"
+        "image": "debasishg/cp-kafka-connect:3.2.2"
```
Comment (deanwampler) [inline repo/packages/C/confluent-connect/7/resource.json]: Can't use your own image location!

Comment (debasishg) [inline repo/packages/C/confluent-connect/7/resource.json]: Will change to `lightbend` soon .. that's why the don't merge part in the title

**Comment by debasishg** [comment]
Changed dockerhub repo to lightbend

**Comment by yuchaoran2011** [comment]
LGTM. Merging.

**Pull request merged**